docs/install/configuration-options.md

```diff
@@ -18,13 +18,13 @@ Number of expired storing records `session history`, values:
 
 If this option is `true`, it will be possible to search users in LDAP.
 
-To use this option you need to install kartik typeahead:
+To use this option, you need to install [yetopen/yii2-usuario-ldap](https://github.com/YetOpen/yii2-usuario-ldap):
 
 ```shell
-composer require kartik-v/yii2-widget-typeahead "dev-master"
+composer require yetopen/yii2-usuario-ldap "*"
 ```
 
-#### ldapUserAttributes (Type: `array`)
+#### LDAPUserAttributes (Type: `array`)
```

Comment:
I meant capitalization in the description, not in the attribute name! We have no config param starting with a capital. Please restore the previous attribute name, and fix the description at line 29